scripts/uninstall.sh

```
@@ -0,0 +1,68 @@
+if available systemctl; then
+    configure_systemd
+fi
+
+if available llama-swap; then
+    status "Removing llama-swap binary..."
+    $SUDO rm $(which llama-swap)
+fi
+
+if [ -f "/usr/share/llama-swap/config.yaml" ]; then
+    while true; do
+        read -p "Delete config.yaml (/usr/share/llama-swap/config.yaml)? [y/N] " answer
```

Shellcheck (0.10.0)

[warning] In POSIX sh, read -p is undefined.

(SC3045)